### PR TITLE
Fix/user navbar model

### PR DESCRIPTION
**Pull request by Tav0**
Proper fix for navbar, because the user model was not being loaded to ember-data.

### CODE REVIEW

**Comment by jnayak1** [comment]
@Tav0 Why are we adding back the "data" level? Is it so we can make an ember User model? If so, why do we access the attributes for the user with `user.data.attributes.name_of_attribute`? If it was in a User model in Ember, wouldn't it just be `user.name_of_attribute`?
I'm probably misunderstanding something so I'll checkout the branch and play with it.

**Comment by jnayak1** [comment]
Also, shouldn't we be getting the User in the route and passing it to the component? Currently we are making the request in the component. That's probably what's causing the navbar to render a bit slowly sometimes compared to the rest of the page.

**Comment by jnayak1** [comment]
https://github.com/CenterForOpenScience/osf-meetings/blob/develop/app/components/meetings-navbar.js#L27
Shouldn't content-type be application/vnd.api+json?

File: meetings/api/migrations/0001_initial.py [hunk 1]
```diff
@@ -20,14 +20,17 @@ class Migration(migrations.Migration):
         migrations.CreateModel(
             name='Conference',
             fields=[
-                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
```
Comment (jnayak1) [inline meetings/api/migrations/0001_initial.py:23]: I don't think we need any migrations in this module. We don't have any models defined in the api module.

**Comment by jnayak1** [comment]
Pass done 👍 .
I would say:
- [ ] Delete api migration file
- [ ] Get user in route and pass it into component

Is this just a quick fix to get the User back in to store so that we can work on other things? Is not having the User in store a blocker to other tickets? If so, we could merge this and then make changes in another PR.

**Comment by Ghalko** [comment]
@Tav0 I agree with https://github.com/CenterForOpenScience/osf-meetings/pull/92#issuecomment-244200085

**Comment by Tav0** [comment]
@jnayak1 data level is because ember is asking for it in other to do the `pushPayload` else we get an error. Before when it was removed the user data was not being populated in ember-data. 

As for the user being get from the router instead of the component. None of that will be used anymore with the new oauth implementation I'm working on. Check my new PR for the follow up. The new PR will replace this PR.

**Pull request closed without merging**